module/VuFind/src/VuFind/Search/Solr/Results.php

```diff
@@ -349,6 +352,21 @@ public function getFacetList($filter = null)
                 $currentSettings['displayText'] = $translate
                     ? $this->translate([$translateTextDomain, $displayText])
                     : $displayText;
+                $currentSettings['displayText'] = $displayText;
```

Comment:
You are overwriting `$currentSettings['displayText']` here -- I believe that the above three lines need to be removed to avoid performing duplicate work.

Reply:
Happy to help!